quarkus-picocli/src/test/java/PicocliDevIT.java

```diff
@@ -0,0 +1,96 @@
+import static java.util.concurrent.CompletableFuture.runAsync;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class PicocliDevIT {
+
+    @QuarkusApplication
+    static final RestService greetingApp = new RestService()
+            .withProperty("quarkus.profile", "dev")
```

Comment:
We have this property in both application.properties and here. Is it really needed?

Reply:
Yes, application.properties active the dev profile with the dev beans needed in the app build. If not beans specific to `dev` profile are not included and commands are unavailable then the tests fail.

Reply:
Proper explanation would be that framework doesn't know that you need the runtime property during the build time, so it only passes it as system property to a Java process.

Reply:
We now have `properties = "dev.properties"` option. Do we still need to add properties explicitly?

Reply:
No, now we didn't need it, thanks @fedinskiy .